chef-cookbooks/yum-epel
# Remove CentOS 6 / Amazon Linux 201X support/testing

Now that EL6 has been EOL'd upstream, we should remove support from this cookbook. In addition, include the following fixes:

- Improve InSpec test by using yum.repo resource
- Fix repo descriptions on Amazon Linux
- Test all supported repos in new "all" suite
- Add various modular and playground repos for EL8
- Update README
- MDL Fixes
- Cleanup metadata.rb formatting

This should be a major version bump for the cookbook.
